src/core/permissions/Permission.js

```diff
@@ -13,23 +13,17 @@ function enumFactory(friendlyName) {
   };
 }
 
-export class Permission extends Enum {}
+export class Permission extends Enum {
```

Comment:
I have something very similar in `loom-data`, though I don't expose it yet, so it might be worth considering switching to that. in that case, this will probably be unnecessary.